File: Digital Business Enablement/bluegather/src/main/java/br/com/fiap/bluegather/dto/MomentoDTO.java
package br.com.fiap.bluegather.dto;

import jakarta.validation.constraints.*;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MomentoDTO {
    private Long id;
    
    @NotBlank(message = "O campo nome não pode estar vazio.")
    private String nome;
}

File: Enterprise Application Development/bluegather/Dtos/MomentoDto.cs
using System.ComponentModel.DataAnnotations;

namespace BlueGather.Dto
{
    public class MomentoDto
    {
        public long? Id { get; set; }

        [Required(ErrorMessage = "O campo nome não pode estar vazio.")]
        public string Nome { get; set; }
    }
}